runtime/shared_common/shrinit.cpp

```diff
@@ -4303,6 +4303,17 @@ j9shr_shutdown(J9JavaVM *vm)
 		if (config->jclCacheMutex) {
 			omrthread_monitor_destroy(config->jclCacheMutex);
 		}
+
+		if (0 != compareAndSwapUDATA((uintptr_t *)&config->romToRamHashTable,
+			(uintptr_t)romToRamHashTable,
+			(uintptr_t)0)
+		) {
```

Comment:
Please move the locals here as requested (it narrow the gap between the read and test of `romToRamHashTable` and there's no need to read `romToRamHashTableMutex` unless we're going to use and clear it).
When a function call is split, each argument goes on a line by itself.

Reply:
> When a function call is split, each argument goes on a line by itself.

See https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line.

Reply:
I fixed it.